views/pages/resources/cheque-requests.pug

```diff
@@ -3,6 +3,9 @@ extends /views/templates/frontend-base.pug
 append styles
     link(href="/assets/css/pages/resources/cheque-requests.css" rel="stylesheet")
 
+append scripts
+    script(src="/assets/js/collapsible.js" defer)
+    

```

Comment:
what is this insertion for

Reply:
it probably should have been there before; the collapsibles on this page weren't working without it.